app/controllers/concerns/solr.rb

```diff
@@ -20,7 +20,29 @@ def set_search_result_arrays(sub = nil)
         # the solr query should be field:"keyword"
         if @search_term.start_with? '"' and @search_term.end_with? '"'
             search_term2 = @search_term.downcase
-        elsif @search_term.include? ' ' or @search_term.downcase.include? ' or ' or @search_term.downcase.include? ' and '
+        elsif @search_term.downcase.include? ' and '
+            s = '('
+            @search_term.downcase.split('and').each_with_index do |t, index|
```

Comment:
@yf508 if we donwcase string before searching for AND this will limit the ability to add search for legitimate "and".

Reply:
???

Reply:
???

Reply:
I'm sorry my first comment was very cryptic. I think we should only support AND and OR as capital (alternatively AND could be replaced with + and OR with ||). So when users enter a phrase such as _Recites the details of the tax grant made by convocation, the exemptions **and** conditions set, **and** the names of the collectors._ the code will not remove **and** from the search phrase. Not sure what difference on a search result will this have thought. However, this is using AND and OR with the notion of common search language syntax.